hotline/pages/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+from django.contrib.flatpages.forms import FlatpageForm
+from django.contrib.sites.models import Site
+from django.utils.translation import ugettext
+
+reserved = ['edit', 'create', 'list', 'delete']
+
+
+class FlatterPageForm(FlatpageForm):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['sites'].initial = Site.objects.first()
```

Comment:
`initial` on fields and forms is ignored when a form is bound, so this can't be a necessary piece of code, right?